releases/v0.2.0.md

```diff
@@ -0,0 +1,71 @@
+# Release Notes for v0.2.0
+Confidential Containers has adopted a six-week release cadence.
+This is our first release on this schedule.
+This release mainly features incremental improvements to our build system and tests
+as well as minor features, adjustments, and cleanup.
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential Containers
+
+## What's new
+
+- Kata CI uses existing Kata tooling to build components
+- Kata CI caches build environments for components
+- Incremental advances in signature support
+- KBS no longer required to use unencrypted images with SEV.
+
+## Hardware Support
+Confidential Containers is tested with attestation on the following platforms:
+- Intel TDX
+- AMD SEV
+
+The following platforms are untested or partially supported:
+- AMD SEV-ES
+- IBM Z SE
+
+The following platforms are in development:
+- Intel SGX
+- AMD SEV-SNP
+
+## Limitations
+
+The following are known limitations of this release:
+
+- Platform support is currently limited, and rapidly changing
+  * S390x is not supported by the CoCo operator
+  * AMD SEV-ES has not been tested.
+  * AMD SEV does not support container image signature validation.
+- Attestation and key brokering support is still under development
+  * The disk-based key broker client (KBC) is used when there is no HW support, but is not suitable for production (except with encrypted VM images).
+  * Currently, there are two KBS that can be used:
+    - simple-kbs:  simple key broker service (KBS) for SEV(-ES).
+    - [Verdictd](https://github.com/inclavare-containers/verdictd): An external project with which Attestation Agent can conduct remote attestation communication and key acquisition via EAA KBC
+  * The full-featured generic KBS and the corresponding KBC are still in the development stage.
+  * For developers, other KBCs can be experimented with.
+  * AMD SEV must use a KBS even for unencrypted images.
+- The format of encrypted container images is still subject to change
+  * The oci-crypt container image format itself may still change
+  * The tools to generate images are not in their final form
+  * The image format itself is subject to change in upcoming releases
+  * Image repository support for encrypted images is unequal
+- CoCo currently requires a custom build of `containerd`
+  * The CoCo operator will deploy the correct version of `containerd` for you
+  * Changes are required to delegate `PullImage` to the agent in the virtual machine
+  * The required changes are not part of the vanilla `containerd`
+  * The final form of the required changes in `containerd` is expected to be different
+  * `crio` is not supported
+- CoCo is not fully integrated with the orchestration ecosystem (Kubernetes, OpenShift)
+  * OpenShift is a non-started at the moment due to their dependency on CRIO
```

Comment:
nit: non-started -> non-starter 
this typo is a leftover from the previous release so if possible better to fix the typo in this release :-)